### PR TITLE
Sidebar 

### PR DESCRIPTION
**Pull request by aatanasova**

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  SIDEBAR_MENU_LOGO: 'sidebar-menu-logo',
+  SIDEBAR_ACTIVE_MENU_ITEM: 'sidebar-active-menu-item',
+  SIDEBAR_ICON_SIZE: 'sidebar-icon-size',
+  SIDEBAR_DISABLED_MENU_ITEM: 'sidebar-disabled-item',
+  SIDEBAR_SUPPORT_BUTTON: 'sidebar-support-button',
+  SIDEBAR_BUTTON_NAME: 'sidebar-button-name',
+  SIDEBAR_NO_ICON_IN_LIST: 'no-icon-in-list'
+};
+
+  const VatRecovery = [
```
Comment (sweko) [inline src/components/Sidebar.js]: Extract to `constants` file

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                            return <li className={CLASS_NAMES.SIDEBAR_NO_ICON_IN_LIST}>
+                                      <a href="#" className={CLASS_NAMES.SIDEBAR_DISABLED_MENU_ITEM}>{item.name}</a>
+                                   </li>;
+                          })}
+                        </ul>
+
+                        <li>
+                          <a href="#" className={CLASS_NAMES.SIDEBAR_ICON_SIZE}> 
+                            <Icon icon='bill.svg' />
+                            Evidance Qualification
```
Comment (sweko) [inline src/components/Sidebar.js]: change spelling

File: src/components/Sidebar.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  SIDEBAR_NO_ICON_IN_LIST: 'no-icon-in-list'
+};
+
+  const VatRecovery = [
+    { name: 'Your Reclaims' }, { name: 'Your Invoices' }, { name: 'Optimize Recovery' }
+  ];
+  const EvidanceQualification = [
+    { name: 'Vendor Direct'}
+  ];
+   const TAXTailor = [
```
Comment (sweko) [inline src/components/Sidebar.js]: Change casing

File: src/stylesheets/sidebar.scss [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+@media (max-width: 767px) {
+  .nav-side-menu {
+    position: relative;
+    width: 100%;
+    margin-bottom: 10px;
+  }
+
+  .sidebar-menu-logo {
+    text-align: left !important;
```
Comment (sweko) [inline src/stylesheets/sidebar.scss]: do we need `!important`

**Review by sweko — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.js (hunk 2 above)
Comment (aatanasova) [reply src/components/Sidebar.js]: Fixed

File: src/components/Sidebar.js (hunk 3 above)
Comment (aatanasova) [reply src/components/Sidebar.js]: Fixed

File: src/stylesheets/sidebar.scss (hunk 4 above)
Comment (aatanasova) [reply src/stylesheets/sidebar.scss]: Fixed

File: src/components/Sidebar.js (hunk 1 above)
Comment (aatanasova) [reply src/components/Sidebar.js]: Fixed

**Review by sweko — APPROVED** [review]

**Pull request merged**